localenv/happy-life-bank/docker-compose.yml

```diff
@@ -64,7 +64,8 @@ services:
       TENANT_ID: cf5fd7d3-1eb1-4041-8e43-ba45747e9e5d
       TENANT_SECRET: iyIgCprjb9uL8wFckR+pLEkJWMB7FJhgkvqhTQR/964=
       GRAPHQL_URL: http://happy-life-bank-backend:3001/graphql
-      WEBHOOK_SIGNATURE_SECRET: webhook_secret
+      WEBHOOK_SIGNATURE_SECRET: iyIgCprjb9uL8wFckR+pLEkJWMB7FJhgkvqhTQR/964=
+      WEBHOOK_SIGNATURE_VERSION: 1
```

Comment:
updating the docker compose file so that the webhooks between backend <> POS service are signed and correctly validated